Data/Graph/Inductive/NodeMap.hs

```diff
@@ -107,6 +125,11 @@ insMapNode m a g =
     let (n, m') = mkNode m a
     in (insNode n g, m', n)
 
+insMapLookupNode :: (Ord a, DynGraph g) => NodeMap a -> a -> g a b -> (Bool, g a b, NodeMap a, LNode a)
```

Comment:
Documentation please.  What is the purpose of this function?

Reply:
Oops yes, I forgotten to document that. It is the function asked in the issue.